vendor/github.com/portworx/torpedo/drivers/scheduler/k8s/k8s.go

```diff
@@ -469,6 +469,8 @@ func validateSpec(in interface{}) (interface{}, error) {
 		return specObj, nil
 	} else if specObj, ok := in.(*networkingv1beta1.Ingress); ok {
 		return specObj, nil
+	} else if specObj, ok := in.(*corev1.Namespace); ok {
```

Comment:
i'm running torpedoUpdate with integration test, for now lets only merge this in master